resolvconf/resolvconf.go

```diff
@@ -52,7 +53,11 @@ type File struct {
 func Get() (*File, error) {
 	resolv, err := ioutil.ReadFile("/etc/resolv.conf")
 	if err != nil {
-		return nil, err
+		if os.IsNotExist(err) {
+			logrus.Infof("/etc/resolv.conf does not exist. Using default external servers : %v", defaultIPv4Dns)
```

Comment:
I have tested that, if `/etc/resolv.conf` is an empty file, then `resolv` will be `[]byte{}`, if `/etc/resolv.conf` does not exist, then `resolv` whill be `nil` (It's ok to read `resolv` if it's `nil`), and no matter if `resolv` is  `nil` or an empty slice, `ioutils.HashData(bytes.NewReader(resolv))` returns a same hash.